tests/cbtest/gtest/test_zrepl_prot.cc

```diff
@@ -1461,22 +1462,88 @@ TEST(DiskReplaceTest, SpareReplacement) {
 	sleep(5);
 }
 
+static void verify_snapshot_details(std::string zvol_name, std::string json) {
+	struct json_object *jobj = NULL, *jarr = NULL;
+	struct json_object *jsnap, *jsnapname, *jprop;
+	int arrlen, i;
+	std::string output;
+	char jsval[32];
+	uint64_t jival;
+	jobj = json_tokener_parse(json.c_str());
```

Comment:
4 pointers have been defined above which are related to the json objects. Where are we freeing them ?
If already freed somewhere in above code then ignore and if possible please add a reference where are you freeing them, else free the pointers at the end.

Reply:
@utkarshmani1997 . Thanks for pointing out this. I will update it.